securesystemslib/keys.py

```diff
@@ -518,6 +519,10 @@ def format_metadata_to_key(key_metadata, hash_algorithms=securesystemslib.settin
        'keyval': {'public': '...',
                   'private': '...'}}
 
+    hash_algorithms:
+      An optional parameter used to provide custom hash algorithms when
+      creating the key dictionary.
+
```

Comment:
One purpose of the docstring is to prevent the reader from having to read the code to understand how the function works and what arguments to pass. Therefore it should tell us the type of the argument, especially where it isn't obvious from the function signature.

```suggestion
    hash_algorithms:
      An optional list containing the names of hash algorithms to use when
      generating keyids. Defaults to securesystemslib.settings.HASH_ALGORITHMS

```

Reply:
Addressed in a follow on commit.